server/service/hosts_test.go

```diff
@@ -523,7 +523,7 @@ func TestHostAuth(t *testing.T) {
 	globalHost := &fleet.Host{}
 
 	ds.AppConfigFunc = func(ctx context.Context) (*fleet.AppConfig, error) {
-		return &fleet.AppConfig{}, nil
+		return &fleet.AppConfig{MDM: fleet.MDM{EnabledAndConfigured: true, WindowsEnabledAndConfigured: true}}, nil
```

Comment:
Looks like this change wasn't needed after all?

Reply:
aha yeah, forgot to revert! fixed!